src/templates/ObservabilityPackDetails.js

```diff
@@ -78,7 +79,37 @@ const ObservabilityPackDetails = ({ data, location }) => {
                 <h3>Description</h3>
                 <p>{pack.description}</p>
               </Tabs.Page>
+              <Tabs.Page id="dependencies">
+                <Intro
+                  css={css`
+                    margin-bottom: 16px;
```

Comment:
should we just add margin bottom to the intro component? seems more likely it would be wanted, and if it was ever going to the the bottom component then someone could specify no margin 🤷‍♀️